docs/Authenticating-Users-via-SAML.md

```diff
@@ -134,8 +132,6 @@ Within portal.properties, make sure that:
 
 Then, modify the section labeled `authentication`. See SAML parameters shown in example below:
 
-    # authentication
```

Comment:
Doesn't this section a line like authentication=saml ?
I'm not sure, but I'm surprised it is not here. It seems like the authentication property is missing from all these sections, so maybe I missed something or am confused.

Ah, I see a reference to setting the method on the command line like this CATALINA_OPTS='-Dauthenticate=saml' (in portal.properties-Reference.md). I think a quick test of the proper authorization when setting the authentication through CATALINA_OPTS under both googleplus and social_auth would be a good idea.

So maybe the documentation now use this as the primary way to set the authentication method.